provisioning/ansible/roles/spark/templates/run-application.sh.j2

```diff
@@ -25,4 +25,4 @@ echo -e "${GREEN}[info] Running application $APPLICATION...${NC}"
 export SPARK_MASTER_HOST={{ masterIP }}
 export SPARK_LOCAL_HOST={{ masterIP }}
 
-{{ dir_spark }}/spark-bin/bin/spark-submit --total-executor-cores {{ spark_worker_cores }} --executor-memory {{ (spark_worker_memory * 0.8)|int }}M $PYFILES $APPLICATION
+{{ dir_spark }}/spark-bin/bin/spark-submit --total-executor-cores {{ groups['sparkSlave'] | length }} --executor-memory 2G $PYFILES $APPLICATION
```

Comment:
The filled in conf  after deployed looks following: 
```
# Dynamic executors allocation
spark.streaming.dynamicAllocation.enabled            true
spark.streaming.dynamicAllocation.minExecutors       2
spark.streaming.dynamicAllocation.maxExecutors       2
spark.streaming.dynamicAllocation.initialExecutors   1
spark.streaming.dynamicAllocation.reserveRate        0.1
spark.cores.max                                      1
spark.executor.memory                                2G
````
Filled in run-application looks like :
```
/opt/spark/spark-bin/bin/spark-submit --total-executor-cores 1 --executor-memory 2G $PYFILES $APPLICATION

```

However an application protocol-statistics seems to get stuck on one job  - see screenshots attached. 

![screenshot_20171108_072749](https://user-images.githubusercontent.com/11919446/32534563-80d84256-c456-11e7-9e0e-9c2dccc36af7.png)
![screenshot_20171108_072809](https://user-images.githubusercontent.com/11919446/32534581-8cedd722-c456-11e7-84bf-6561464beee1.png)

* The  `--total-executor-cores 1`  might be the issue. When increased to 2, the app worked